plugins/source/aws/resources/services/fsx/volumes.hcl

```diff
@@ -0,0 +1,110 @@
+//check-for-changes
+service          = "aws"
+output_directory = "."
+add_generate     = true
+
+description_modifier "remove_read_only" {
+  words = ["  This member is required."]
+}
+
+resource "aws" "fsx" "volumes" {
+  path = "github.com/aws/aws-sdk-go-v2/service/fsx/types.Volume"
+  ignoreError "IgnoreAccessDenied" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.IgnoreAccessDeniedServiceDisabled"
+  }
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.DeleteAccountRegionFilter"
+  }
+  multiplex "AwsAccountRegion" {
+    path   = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ServiceAccountRegionMultiplexer"
+    params = ["fsx"]
+  }
+  options {
+    primary_keys = ["arn"]
+  }
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource"
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSAccount"
+    }
+  }
+  userDefinedColumn "region" {
+    type        = "string"
+    description = "The AWS Region of the resource"
+    resolver "resolveAWSRegion" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSRegion"
+    }
+  }
+
+  column "resource_arn" {
+    rename = "arn"
+  }
+
+  column "tags" {
+    type = "json"
+    resolver "resolveTags" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveTags"
+    }
+  }
+
+  column "administrative_actions" {
+    skip = true
+  }
```

Comment:
Why are we skipping these fields?

Reply:
Since this PR adds more of the information how about we do `administrative_actions` in a follow up PR?
I'll keep the issue open with a mention to this column